### DISCUSSION TITLE
Creating a "Last Updated" field in tabular view of github project

### DISCUSSION THREAD

**Discussion by karkir0003** [community/community#66629] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Projects

### Body

Hey team. I'm trying to create a new column called "Last Updated" that lives in a table view of a github project. I want this "Last updated" field to update automatically for a given github issue if:

* Issue is edited (ie: updating issue description, adding a comment on issue, linking PR, etc)
* Issue status is changed/dragged within the board view of project (eg: issue gets updated from todo status to done status)

Could someone link a guide or example step by step on configuring such functionality

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by martianmusk** [comment]
+1, any guidance or alternative solutions?

**Comment by rockliffe** [comment]
I would really like this feature too.
